### ISSUE TITLE
Auto-responder (Vacation message), TO DATE

### ISSUE THREAD

**Issue by flaviozluca** [polarismail/Webmail#14]
I found that, if I add an auto-responder via webmail, the TO DATE is blank by default. Then you save it, and it sets the TO date to today´s date, after you save it.

**Comment by polarismail** [comment]
yes, this is intended behavior when the user doesn't set a date.

**Comment by flaviozluca** [comment]
I believe that rarely a customer will set an auto-responder that will last just 1 day.

IMHO, I'd prefer some other action if the TO DATE is blank. An error "fill out the to date" or even setting the to date as indefinite. I wouldn't set it to today's date.

Even an easy way out, leave the "no end date checkbox" marked before saving, by default have it active.

It would make more sense.

Thanks.

**Comment by polarismail** [comment]
why would he mind if it's one day when he sets no date at all ?
has anyone complained about this being an issue ?

**Comment by flaviozluca** [comment]
It's a question of usability. No, nobody complained (which, imho doesn't mean much, since it's a new feature, most don't even use it via webmail yet, etc.).

I asked my colleague who is a developer too and he agreed. It's just not the ideal, in our opinion. It's very unusual to set an auto-responder for 1 single day. I set an auto-responder forever or if Im going on vacation. Not for a single day.

It's just not the ideal way, in our opinion. There are better alternatives than that approach, as I explained.

Easy way out, just keep the checkbox "no end date" marked by default. Makes much more sense.

Or show an error "you didn't choose the end date".

Why set today's date if 99% of the time people don't set an autoresponder for a single day?

Anyway, if you think it's Ok like this, no problem. At least the person probably sees that the system set TO to today's date and can adjust... Or not and the not ideal usability, of course, initiated by user's fault for not choosing an end date, can lead to a configuration that won't work as he desires.

If we can avoid that and have better usability, why not?

My 2 cents, friend, but you decide, of course.  Prefer to keep it that way, no problem.

Thanks.
